Cargo.lock

```diff
@@ -27,6 +27,18 @@ dependencies = [
  "generic-array",
 ]
 
+[[package]]
```

Comment:
That's like a loooot of dependencies added here, do we know why we need so much stuff ?

Reply:
Seems to be caused by `zbus`: https://github.com/Scille/parsec-cloud/pull/6564/files#diff-13ee4b2252c9e516a0547f2891aa2105c3ca71c6d7a1e682c69be97998dfc87eR4838-R4877

Reply:
`zbus` is required by `secret-service`

Reply:
We could tweak the used features of `keyring` to remove some packages: https://github.com/hwchen/keyring-rs/blob/252f43c3e7453e4099c4eb99fa3871e90d2c8579/Cargo.toml#L15-L31

We currently use `platform-all`, but we can be more peeky about that

Reply:
It seems zbus 4 may improve things here, https://github.com/hwchen/secret-service-rs/issues/71#issue-2130430440

I guess we don't have much choice right now, so we'll just leave it as is for the moment ;-)